neptune/new/internal/utils/traceback_job.py

```diff
@@ -31,18 +31,21 @@
 
 class TracebackJob(BackgroundJob):
 
-    def __init__(self, path: str):
+    def __init__(self, path: str, fail_on_exception: bool = True):
         self._uuid = uuid.uuid4()
         self._started = False
         self._path = path
+        self._fail_on_exception = fail_on_exception
 
     def start(self, run: 'Run'):
         if not self._started:
             path = self._path
+            fail_on_exception = self._fail_on_exception
```

Comment:
What is it for?

Reply:
because you can't reference self easilsy from a `def` function passed as lambda